src/assets/configuration/config.json

```diff
@@ -8,25 +8,25 @@
 	"domains": ["workgroup"],
 	"itemsPerPage": 10,
 	"mams": [{
-		"casEndpoint": "http://<address-of-cas-server-here-relative-to-mams-service>:8082/ICinegyDataAccessService",
-		"dbName": "CinegyDatabaseNameHere",
-		"dbServer": "NameOfDatabaseServerHere",
-		"mamEndpoint": "https://<address-of-mams-hosting-or-routing-server-here>/mams/1.0/api/mam/",
```

Comment:
Please don't submit changes to the example config file - best to leave our model 'example' values in place, and just keep local changes uncommitted.